style.css

```diff
@@ -18,9 +18,6 @@ header {
   justify-content: space-between;
   align-items: center;
   text-align: center;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, there are a few repeatings rules that need to be cleaned. For example;
```
.nav-container, .mainpage {
  display: flex;
  align-items: center;
  text-align: center;
}
```